rln/www/index.js

```diff
@@ -0,0 +1,108 @@
+import * as rln from "rln";
+import * as wc from "./witness_calculator"
+import * as files from './files'
+
+rln.init_panic_hook();
+
+function _base64ToArrayBuffer(base64) {
+    var binary_string = window.atob(base64);
+    var len = binary_string.length;
+    var bytes = new Uint8Array(len);
+    for (var i = 0; i < len; i++) {
+        bytes[i] = binary_string.charCodeAt(i);
+    }
+    return bytes;
+}
+
+function _concatenate(resultConstructor, ...arrays) {
+    let totalLength = 0;
+    for (const arr of arrays) {
+        totalLength += arr.length;
+    }
+    const result = new resultConstructor(totalLength);
+    let offset = 0;
+    for (const arr of arrays) {
+        result.set(arr, offset);
+        offset += arr.length;
+    }
+    return result;
+}
+
+
+    function serializeMessage(uint8Msg, memIndex, epoch, idKey) {
+        // calculate message length
+        const msgLen = Buffer.allocUnsafe(8);
+        msgLen.writeUIntLE(uint8Msg.length, 0, 8);
+
+        // Converting index to LE bytes
+        const memIndexBytes = Buffer.allocUnsafe(8)
+        memIndexBytes.writeUIntLE(memIndex, 0, 8);
+
+        // [ id_key<32> | id_index<8> | epoch<32> | signal_len<8> | signal<var> ]
+        return _concatenate(Uint8Array, idKey, memIndexBytes, epoch, msgLen, uint8Msg )
+    }
+
+const verifKeyUint8Array = _base64ToArrayBuffer(files.verification_key);
+const zkeyUint8Array = _base64ToArrayBuffer(files.zkey);
+const circomUint8Array = _base64ToArrayBuffer(files.rln_wasm);
```

Comment:
Should a dev be able to specify the depth and the zkey and verification key to use? or should these be embedded in the code?